dataplanes/vpp/cmd/dataplanetestclient/dataplane-test-client.go

```diff
@@ -139,115 +141,140 @@ func main() {
+	//		testName: "all good",
+	//		localSource: &common.LocalMechanism{
+	//			Type: common.LocalMechanismType_KERNEL_INTERFACE,
+	//			Parameters: map[string]string{
+	//				nsmvpp.NSMkeyNamespace:        srcNamespace,
+	//				nsmvpp.NSMkeyIPv4:             "2.2.2.2",
+	//				nsmvpp.NSMkeyIPv4PrefixLength: "24",
+	//			},
+	//		},
+	//		localDestination: &dataplaneapi.Connection_Local{
+	//			Local: &common.LocalMechanism{
+	//				Type: common.LocalMechanismType_KERNEL_INTERFACE,
+	//				Parameters: map[string]string{
+	//					nsmvpp.NSMkeyNamespace:        dstNamespace,
+	//					nsmvpp.NSMkeyIPv4:             "2.2.2.3",
+	//					nsmvpp.NSMkeyIPv4PrefixLength: "24"},
+	//			},
+	//		},
+	//		shouldFail: false,
+	//	},
+	//	{
+	//		testName: "missing source namespace",
+	//		localSource: &common.LocalMechanism{
+	//			Type: common.LocalMechanismType_KERNEL_INTERFACE,
+	//			Parameters: map[string]string{
+	//				// nsmutils.NSMkeyNamespace:        srcNamespace,
+	//				nsmvpp.NSMkeyIPv4:             "2.2.2.2",
+	//				nsmvpp.NSMkeyIPv4PrefixLength: "24",
+	//			},
+	//		},
+	//		localDestination: &dataplaneapi.Connection_Local{
+	//			Local: &common.LocalMechanism{
+	//				Type: common.LocalMechanismType_KERNEL_INTERFACE,
+	//				Parameters: map[string]string{
+	//					nsmvpp.NSMkeyNamespace:        dstNamespace,
+	//					nsmvpp.NSMkeyIPv4:             "2.2.2.3",
+	//					nsmvpp.NSMkeyIPv4PrefixLength: "24"},
+	//			},
+	//		},
+	//		shouldFail: true,
+	//	},
+	//	{
+	//		testName: "source has ip, but destination doesn't",
+	//		localSource: &common.LocalMechanism{
+	//			Type: common.LocalMechanismType_KERNEL_INTERFACE,
+	//			Parameters: map[string]string{
+	//				nsmvpp.NSMkeyNamespace:        srcNamespace,
+	//				nsmvpp.NSMkeyIPv4:             "2.2.2.2",
+	//				nsmvpp.NSMkeyIPv4PrefixLength: "24",
+	//			},
+	//		},
+	//		localDestination: &dataplaneapi.Connection_Local{
+	//			Local: &common.LocalMechanism{
+	//				Type: common.LocalMechanismType_KERNEL_INTERFACE,
+	//				Parameters: map[string]string{
+	//					nsmvpp.NSMkeyNamespace: dstNamespace,
+	//					// nsmutils.NSMkeyIPv4:             "2.2.2.3",
+	//					nsmvpp.NSMkeyIPv4PrefixLength: "24"},
+	//			},
+	//		},
+	//		shouldFail: true,
+	//	},
+	//	{
+	//		testName: "wrong prefix length",
+	//		localSource: &common.LocalMechanism{
+	//			Type: common.LocalMechanismType_KERNEL_INTERFACE,
+	//			Parameters: map[string]string{
+	//				nsmvpp.NSMkeyNamespace:        srcNamespace,
+	//				nsmvpp.NSMkeyIPv4:             "2.2.2.2",
+	//				nsmvpp.NSMkeyIPv4PrefixLength: "34",
+	//			},
+	//		},
+	//		localDestination: &dataplaneapi.Connection_Local{
+	//			Local: &common.LocalMechanism{
+	//				Type: common.LocalMechanismType_KERNEL_INTERFACE,
+	//				Parameters: map[string]string{
+	//					nsmvpp.NSMkeyNamespace:        dstNamespace,
+	//					nsmvpp.NSMkeyIPv4:             "2.2.2.3",
+	//					nsmvpp.NSMkeyIPv4PrefixLength: "24"},
+	//			},
+	//		},
+	//		shouldFail: true,
+	//	},
+	//}
```

Comment:
If these lines are going to be permanently commented out, should we remove them?